spec/support/vcr_cassettes/mpi/find_candidate/orch_search_with_attributes.yml

```diff
@@ -1,93 +1,12 @@
 ---
 http_interactions:
 - request:
-    method: POST
-    uri: http://www.example.com/
+    method: post
+    uri: "<MPI_URL>"
     body:
       encoding: ASCII-8BIT
-      base64_string:
```

Comment:
The actual diff is that blank middle name is no longer adding a second `<given>` node
<img width="1160" alt="Screen Shot 2021-09-14 at 10 03 12 PM" src="https://user-images.githubusercontent.com/19188/133358419-26e7f0c3-9920-496d-a712-cd88800a9bef.png">